packages/microsite/package.json

```diff
@@ -50,7 +52,6 @@
     "cosmiconfig": "^7.0.0",
     "esbuild": "^0.8.16",
     "execa": "^4.0.3",
-    "globby": "^11.0.1",
```

Comment:
Unless I'm mistaken, I think `globby` needs to stay here. I think it got hoisted into the root? There is some logic in the `cli` that uses it.

Reply:
Ah you're right! I'll add it back. And I'll look at adding a regression test so I don't make the same mistake again! :)

Incidentally, this is something that [PNPM](https://pnpm.js.org/) would catch. I've been meaning to investigate switching my projects over from Yarn since it's a dead end for me (they apply semi-mysterious patches to some dependencies for their Plug'N'Play alternative to `node_modules`, even if you opt out of PnP, which is also a security concern in my opinion).

Reply:
Welp, glad I wanted to write a real non-example test. Turns out this setup doesn’t support tsconfig `paths`, which is probably gonna cause some pain with this directory structure. So I’m also looking into the solve for that.

Reply:
Ah that would be a problem, yeah. Let me know if you get stuck.